testing/cli/test_cli_sct_register_to_template.py

```diff
@@ -125,3 +144,29 @@ def test_sct_register_to_template_mismatched_xforms(tmp_path, capsys):
                                             '-s', sct_test_path('t2', 't2_seg-manual.nii.gz'),
                                             '-l', sct_test_path('t2', 'labels.nii.gz')])
     assert "Image sform does not match qform" in capsys.readouterr().out
+
+
+def test_sct_register_to_template_3_labels(tmp_path, labels_discs):
+    """Test registration with 3 labels."""
```

Comment:
I'm a bit confused by the name/description of this test. It says "3 labels", but `labels_discs` contains 4 disc labels? 

From the context in this PR, it seems like this is meant to test the ">2 labels" case, so perhaps we could update it like this:

```suggestion
def test_sct_register_to_template_more_than_2_labels(tmp_path, labels_discs):
    """
    Test registration with >2 labels. This test (and the custom disc label file) are needed because the existing
    `t2/labels.nii.gz` file only contains 2 labels. But, registration will be performed differently depending on
    whether there are 1, 2, or >2 labels. See also:

    https://spinalcordtoolbox.com/user_section/tutorials/vertebral-labeling/how-many-labels-for-registration.html
    """
```

Reply:
> I'm a bit confused by the name/description of this test. It says "3 labels", but labels_discs contains 4 disc labels?

sorry about that-- i started doing it with 3 labels, and then added one. Do you mind fixing the name of the function and the docstrings? thanks!!